biopandas/mmcif/pandas_mmcif.py

```diff
@@ -68,6 +69,58 @@ def read_mmcif(self, path):
         # self.header, self.code = self._parse_header_code() #TODO: implement
         self.code = self.data["entry"]["id"][0].lower()
         return self
+    
+    def get_model(self, model_index: int) -> PandasMmcif:
+        """Returns a new PandasMmcif object with the dataframes subset to the
+        given model index.
+
+        Parameters
+        ----------
+        model_index : int
+            An integer representing the model index to subset to.
+
+        Returns
+        ---------
+        pandas_pdb.PandasPdb : A new PandasMMcif object containing the
+            structure subsetted to the given model.
+        """
+
+        df = copy.deepcopy(self)
+        breakpoint()
+        if "ATOM" in df.df.keys():
+            df.df["ATOM"] = df.df["ATOM"].loc[df.df["ATOM"]["pdbx_PDB_model_num"] == model_index]
+        if "HETATM" in df.df.keys():
+            df.df["HETATM"] = df.df["HETATM"].loc[
+                df.df["HETATM"]["pdbx_PDB_model_num"] == model_index
+            ]
+        return df
+
+    def get_models(self, model_indices: List[int]) -> PandasMmcif:
+        """Returns a new PandasMmcif object with the dataframes subset to the
+        given model index.
+
+        Parameters
+        ----------
+        model_indices : List[int]
+            A list representing the model indexes to subset to.
+
+        Returns
+        ---------
+        pandas_pdb.PandasMmtf : A new PandasMmcif object
+            containing the structure subsetted to the given model.
+        """
+
+        df = copy.deepcopy(self)
+
+        if "ATOM" in df.df.keys():
```

Comment:
as above

Reply:
Took that pattern from the pandas_pdb and pandas_mmtf file, but can change all three of them

Reply:
named it `atomic_df` for now to adhere with patterns in the source code

Reply:
I wouldn't use `df`, since it's a `PandasMmcif` object. Maybe: `pmmcif`

Reply:
fair, tried to keep it generic so the method can be the same in the three different file objects. Maybe `pandas_object`? Or `pandas_structure`?

Reply:
Either is fine but I'd opt for `biopandas` over `pandas` to make this distinction clear.